server/app.js

```diff
@@ -11,29 +11,24 @@ const app = express();
 
 // view engine setup
 app.set("views", path.join(__dirname, "views"));
-app.set("view engine", "ejs");
+app.set("view engine", "jade");
 
 app.use(cors());
 app.use(logger("dev"));
 app.use(express.json());
-app.use(express.urlencoded({extended: false}));
+app.use(express.urlencoded({ extended: false }));
 app.use(cookieParser());
-app.use(express.static(path.join(__dirname, "../client/build")));
 app.use(express.static(path.join(__dirname, "public")));
 
-app.use("/api/v1/map", mapRouter);

```

Comment:
All the routing changes in this file should be reverted so the frontend is server properly